pages/rides-overview/index.js

```diff
@@ -0,0 +1,16 @@
+import { rides } from "../../lib/rides";
+import React from "react";
+import { StyledCard } from "../../components/Card/Card.styled";
```

Comment:
Your are not using this component for this overview, right? Then you can remove the import here.